src/Pulp/Watch.purs

```diff
@@ -27,19 +28,19 @@ import Pulp.System.FFI
 import Pulp.System.TreeKill (treeKill)
 import Pulp.Outputter
 
+import Unsafe.Coerce
+
 foreign import watch ::
   Array String
   -> (String -> EffN Unit)
   -> EffN Unit
 
--- launchAff's type in Control.Monad.Aff changed to forall e a. Aff e a -> Eff (err :: EXCEPTION) Unit
--- which will cause a duplicate label in row error, here.  So use the old version for now.
-launchAff :: forall e a. Aff e a -> Eff e (Canceler e)
-launchAff = runAff (const (pure unit)) (const (pure unit)) 
+removeErrLabel :: forall e a. Aff (err :: EXCEPTION | e) a -> Aff e a
+removeErrLabel = unsafeCoerce
 
-watchAff :: forall e. Array String -> (String -> AffN Unit) -> AffN Unit
+watchAff ::  Array String -> (String -> AffN Unit) -> AffN Unit
 watchAff dirs callback =
-  liftEff (watch dirs (\path -> launchAff (callback path) >>= \_ -> pure unit))
+  liftEff (watch dirs (\path -> launchAff (removeErrLabel $ callback path) >>= \_ -> pure unit))
```

Comment:
Just two more little things and then I think this looks good to merge:

* Could you please replace both instances of `pure unit` like this with `void`? I think here we could use `watch dirs (void <<< launchAff <<< removeErrLabel <<< callback)`?
* It seems like `runAff` is now imported but not used. When you compile this, does it cause a warning? I would have expected it to.